Resources/Prototypes/RPD/rpd.yml

```
@@ -0,0 +1,215 @@
+  collisionMask: Impassable
+  rotation: User
+  fx: EffectRCDConstruct0
+
+- type: rcd
+  id: Radiator
+  category: AtmosphericUtility
+  sprite: /Textures/Interface/Radial/RPD/radiator.png
+  mode: ConstructObject
+  prototype: HeatExchanger
+  cost: 1
+  delay: 0 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)